### ISSUE TITLE
Focal Loss in Keras

### ISSUE THREAD

**Issue by mkocabas** [unsky/focal-loss#8]
Hi,
Is there any implementation of Focal Loss in Keras or an ongoing effort? I really appreciate cooperation in such a work, please e-mail me if so.

**Comment by mkocabas** [comment]
I've implemented focal loss in keras, you can check it:
[https://github.com/mkocabas/focal-loss-keras](https://github.com/mkocabas/focal-loss-keras)

**Comment by abhisheksgumadi** [comment]
Hi @mkocabas thanks for the reference. How can I use this loss function in case of multi label classification? I understand your implementation in case of a binary classification. What about multi label classification?

**Comment by alyato** [comment]
@abhisheksgumadi 
i think you can check the [https://github.com/keras-team/keras/issues/741#issuecomment-392725919](url)
Thanks

**Comment by chineseqsc** [comment]
@alyato  404 error...

**Comment by alyato** [comment]
@chineseqsc if you click the link directly, it would show you the error.
you can copy the link and open another tab in your chrome.
